.github/ISSUE_TEMPLATE/new-ontology.yml

```diff
@@ -0,0 +1,249 @@
+  - type: input
+    id: contact-github
+    attributes:
+      label: Contact GitHub Username
+      description: What is the GitHub username of the contact for the ontology? Do not include the @ symbol.
+      placeholder: ex. cthoyt
+    validations:
+      required: true
+  - type: input
+    id: contact-orcid
+    attributes:
+      label: Contact ORCID Identifier
+      description: What is the ORCID identifier of the contact for the ontology? ORCID identifiers can be made at https://orcid.org.
+      placeholder: ex. 0000-0003-4423-4370
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        # Artifacts
+  - type: checkboxes
+    id: formats
+    attributes:
+      label: Formats
+      description: Under which formats is your ontology available? See also OBO Principle [FP-002 "Common Format"](https://obofoundry.org/principles/fp-002-format.html).
+      options:
+        - label: OWL RDF/XML (.owl)
+          required: true
+        - label: OBO (.obo)
+          required: false
+        - label: OBO Graph JSON (.json)
+          required: false
+
+  - type: markdown
+    attributes:
+      value: |
+        # Community
+
+        This section of the ontology request form contains information about how this ontology relates to the greater community
+  - type: textarea
+    id: dependencies
+    attributes:
+      label: Dependencies
+      description: |
+        Please provide a list of ontologies that yours depends on using their OBO Foundry prefixes
+      placeholder: |
+        - ENVO
+        - CHEBI
+        - ExO
+    validations:
+      required: false
+  - type: textarea
+    id: related
+    attributes:
+      label: Related
+      description: |
+        Please provide a list of related OBO Foundry ontologies
+    validations:
+      required: false
+  - type: textarea
+    id: usages
+    attributes:
+      label: Usages
+      description: |
+        Please provide a list of current uses for your ontology in YAML format as a list of objects with `user`, `description`, and `example` keys like in the following example:
+        
+        ```yaml
+        - user: https://www.rhea-db.org/
+          description: Rhea uses CHEBI to annotate reaction participants
+          examples:
+            - url: https://www.rhea-db.org/searchresults?q=CHEBI:29748
+              description: "Query for all usages of CHEBI:29748 (chorismate)"
+        - user: http://zfin.org
+          description: ZFIN uses CHEBI to annotate experiments
+          examples:
+            - url: http://zfin.org/action/expression/experiment?id=ZDB-EXP-190627-10
+              description: "A curated zebrafish experiment involving exposure to (5Z,8Z,14Z)-11,12-dihydroxyicosatrienoic acid (CHEBI:63969)"
+        ```
+      placeholder: |
+        ```yaml
+        # YAML goes here
+        ```
+    validations:
+      required: false
+
+  - type: markdown
+    attributes:
+      value: |
+        # Diligence
```

Comment:
Not sure about this title.. I would have difficulties to understand what it means in this context.. Any better idea?

Reply:
Maybe just "Final Checklist"?

Reply:
Or "Other"? "Final Checklist" is fine as well.

Reply:
Just updated this.